examples/high_frequency/emc/double_pulse_test.py

```diff
@@ -0,0 +1,382 @@
+# # Double Pulse Test
+#
+# Description: build the equivalent circuit of a DPT
+#
+# Most examples can be described as a series of steps that comprise a workflow.
+# 1. Import packages and instantiate the application.
+# 2. Insert Circuit Components
+# 3. Create the Wiring
+# 4. Insert simulation set up
+# 3. View the results.
+#
+# Keywords: **Power Electronics**, **Double Pulse Testing**
+
+# ## Perform imports and define constants
+#
+# Perform required imports.
+
+import os
+import tempfile
+import time
+
+import ansys.aedt.core
+
+# Define constants.
+
+AEDT_VERSION = "2024.2"
+NUM_CORES = 4
+NG_MODE = False  # Open AEDT UI when it is launched.
+
+# ## Create temporary directory
+#
+# Create a temporary directory where downloaded data or
+# dumped data can be stored.
+# If you'd like to retrieve the project data for subsequent use,
+# the temporary folder name is given by ``temp_folder.name``.
+
+temp_folder = tempfile.TemporaryDirectory(suffix=".ansys")
+
+# ## Launch AEDT and Circuit
+#
+
+project_name = os.path.join(temp_folder.name, "my_project.aedt")
+
+circuit = ansys.aedt.core.Circuit(
+    project=project_name,
+    version=AEDT_VERSION,
+    non_graphical=NG_MODE,
+    new_desktop=True,
+)
+circuit.modeler.schematic.schematic_units = "mil"
+
+# ## Preprocess
+#
+# Initialize dictionaries that contain all the definitions for the design variables.
+
+des_properties = {
+    "VoltageDCbus": "400V",
+    "r_g1": "2.2",
+    "r_g2": "2.2",
+    "c_dc_link": "0.0005farad",
+    "l_load": "7.7e-5",
+    "Vgate_top": "-5V",
+    "r_load": "0.1",
+    "r_dc_link": "1e6",
+    "v_pwl_high": "18.0V",
+    "v_pwl_low": "-5.0V",
+}
+
+# Define design variables from the created dictionaries.
+for k, v in des_properties.items():
+    circuit[k] = v
+
+# ## Insert Circuit Elements into the Schematic
+#
+y_upper_pin = 5200
+y_lower_pin = 2000
+time_list_pwl = [
+    0.0,
+    5.0e-6,
+    5.001e-6,
+    9.825e-06,
+    9.826e-06,
+    1.1e-05,
+    1.1001e-05,
+    1.3e-05,
+    1.3001e-05,
+]
+volt_list_pwl = [
+    "v_pwl_low",
+    "v_pwl_low",
+    "v_pwl_high",
+    "v_pwl_high",
+    "v_pwl_low",
+    "v_pwl_low",
+    "v_pwl_high",
+    "v_pwl_high",
+    "v_pwl_low",
+]
+
+# Add circuit components to the schematic.
```

Comment:
@tizianrot add a blank line between a markdown cell and a code cell otherwise Jupyter intends this a single cell.